### PR TITLE
Service Refactoring: Change API endpoint & apply amplify logger

### PR DESCRIPTION
**Pull request by chloe-codes1**
# Description

Service Refactoring: Change API endpoint & apply amplify logger

# Checklist

- [x] : Change API endpoint
  - `https://dev.kendra-btns.whatilearened.today/noauth/graphql`
- [x] : Apply amplify logger  - [docs](https://docs.amplify.aws/lib/utilities/logger/q/platform/js)
- [x] : Refactor search related components

### CODE REVIEW

**Review by geoseong — COMMENTED** [review]
actually you don't have to change the code in `kendra-button-front` directory because it's not running in this branch. but it's okay to change this. please be aware of that.

File: frontend/kendra-button-service/kendra.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import { Logger } from 'aws-amplify';
```
Comment (geoseong) [inline frontend/kendra-button-service/kendra.js:1]: This file is vanilla script so please don't import this Logger and remove other related `Logger` code.

**Review by geoseong — CHANGES_REQUESTED** [review]

File: frontend/kendra-button-service/kendra.js (hunk 1 above)
Comment (chloe-codes1) [reply frontend/kendra-button-service/kendra.js:1]: Thank you for letting me know. I'm on it!

**Comment by geoseong** [comment]
Please refactor `Search` related components refer to [this commit](https://github.com/awskrug/kendra-button/commit/e514e4f82cd41f9f9f6ef7105bb8a9ec92aa718b).

**Review by geoseong — APPROVED** [review]
Great work!

**Pull request merged**